tests/acceptance/features/coreApiSharePublicLink1/createPublicLinkShare.feature

```diff
@@ -463,91 +335,32 @@ Feature: create a public link share
-    And parameter "shareapi_enforce_expire_date" of app "core" has been set to "yes"
-    And user "Alice" has uploaded file with content "Random data" to "/randomfile.txt"
-    When user "Alice" creates a public link share using the sharing API with settings
-      | path | randomfile.txt |
-    Then the OCS status code should be "<ocs_status_code>"
-    And the HTTP status code should be "<http_status_code>"
-    And the fields of the last response to user "Alice" should include
-      | file_target | /randomfile.txt |
-      | path        | /randomfile.txt |
-      | item_type   | file            |
-      | share_type  | public_link     |
-      | permissions | read            |
-      | uid_owner   | %username%      |
-      | expiration  | +7 days         |
-    When user "Alice" gets the info of the last public link share using the sharing API
-    Then the OCS status code should be "<ocs_status_code>"
-    And the HTTP status code should be "<http_status_code>"
-    And the fields of the last response to user "Alice" should include
-      | file_target | /randomfile.txt |
-      | path        | /randomfile.txt |
-      | item_type   | file            |
-      | share_type  | public_link     |
-      | permissions | read            |
-      | uid_owner   | %username%      |
-      | expiration  | +7 days         |
-    And the public should be able to download the last publicly shared file using the old public WebDAV API without a password and the content should be "Random data"
-    And the public should be able to download the last publicly shared file using the new public WebDAV API without a password and the content should be "Random data"
-
-    @notToImplementOnOCIS @issue-ocis-2079
-    Examples:
-      | ocs_api_version | ocs_status_code | http_status_code |
-      | 1               | 100             | 200              |
-      | 2               | 200             | 200              |
-
   @issue-ocis-reva-199
-  Scenario Outline: Delete a folder that has been publicly shared and try to access using the public WebDAV API
+  Scenario: Delete a folder that has been publicly shared and try to access using the public WebDAV API
     Given user "Alice" has created folder "PARENT"
     And user "Alice" has uploaded file with content "Random data" to "/PARENT/parent.txt"
     And user "Alice" has created a public link share with settings
       | path        | PARENT |
       | permissions | read   |
-    When user "Alice" deletes folder "PARENT" using the WebDAV API
-    And the public download of file "/parent.txt" from inside the last public link shared folder using the <public-webdav-api-version> public WebDAV API should fail with HTTP status code "404"
+    And user "Alice" has deleted folder "PARENT"
+    When the public download of file "/parent.txt" from inside the last public link shared folder using the new public WebDAV API should fail with HTTP status code "404"
```

Comment:
```suggestion
    When user "Alice" deletes folder "PARENT" using the WebDAV API
    Then the public download of file "/parent.txt" from inside the last public link shared folder using the new public WebDAV API should fail with HTTP status code "404"
```

otherwise the scenario ends with a "When" step, and it is a bit strange to read.